ffmpeg/lpms_ffmpeg.c

```diff
@@ -751,14 +751,14 @@ int process_out(struct input_ctx *ictx, struct output_ctx *octx, AVCodecContext
 
 #define MAX_OUTPUT_SIZE 10
 
-int lpms_transcode(input_params *inp, output_params *params, int nb_outputs)
+int lpms_transcode_and_count(input_params *inp, output_params *params, int nb_outputs, int64_t *framecount, int64_t *pixelcount)
```

Comment:
Can the original `lpms_transcode` API name be retained? Don't think we need to retain back-compat with this (internal) API.

Can we have a `struct transcode_result` output param, rather than separate params for each field? That way we can add new fields without also modifying the function signature (we might have to do that for https://github.com/livepeer/go-livepeer/issues/997)

Reply:
You seem to be contradicting yourself within this one comment.   This is an internal function and we do not need to have an extensible result.  I'd rather not wrap a C structure in go cgo decoration just for this purpose.

...although I see your point, I'd just rather not.   If this is really your preference, please say it a second time and I will give in ;-) .   BTW, this comment doesnt show on all of my logins.  Strange...

Reply:
>  I'd rather not wrap a C structure in cgo decoration

Not sure what you mean here - passing in a Golang data structure to C? If so, agreed that'd be crazy.

Define the result struct in C, something like: `struct transcode_result { int64 frames, pixels; };`, then convert that to a Golang `TranscodeResult` within the `Transcode3` function.